packages/terra-form-select/tests/jest/SelectFieldsMaxHeight.test.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
```

Comment:
It's a little strange to organize these changes into a combined spec file for only max height. Traditionally these would be  split out for each field variant and non-specific to any props/functionality.

/jest/ComboboxField.test.jsx
/jest/MultiSelectField.test.jsx
/jest/TagSelectField.test.jsx
/jest/SearchSelectField.test.jsx

It's not harmful to keep these tests in this file. It's a little less organized but the functionality is still being tested.

Reply:
I was debating about that, I agree this is better approach. Updated test cases